astropy/coordinates/tests/test_celestial_transformations.py

```diff
@@ -5,11 +5,26 @@
 
 from astropy import units as u
 from astropy.coordinates import (
-    CartesianDifferential, CartesianRepresentation, EarthLocation, SkyCoord,
-    galactocentric_frame_defaults)
+    CartesianDifferential,
+    CartesianRepresentation,
+    EarthLocation,
+    SkyCoord,
+    galactocentric_frame_defaults,
+)
 from astropy.coordinates.builtin_frames import (
-    CIRS, FK4, FK5, GCRS, HCRS, ICRS, LSR, FK4NoETerms, Galactic, GalacticLSR, Galactocentric,
-    Supergalactic)
+    CIRS,
+    FK4,
+    FK5,
+    GCRS,
+    HCRS,
+    ICRS,
+    LSR,
+    FK4NoETerms,
```

Comment:
Just curious, is it normal that these are not sorted alphabetically so that eg FK4 is just before FK4NoETerms?

Reply:
@nstarman do you know why?

Reply:
I also noticed another unusual sorting:
```
    SIP,
    Chebyshev1D,
    Chebyshev2D,
    Hermite1D,
   ...
```

Reply:
Hm. `isort` puts classes before functions. I would have guessed that it's getting confused and thinking ``FK4NoETerms`` is a function because of the ``format_doc`` decorator, but ``FK4`` has that as well.

Reply:
`modeling` also appears to suffer from this problem...

Reply:
Sans manual digging, that was my one and only guess. Pragmatically, if it's consistent between applications of ``pre-commit run -a`` then I'm inclined to continue with this PR. If we can understand and fix the sort, even better, but if not, when someone has the spare hours and the inclination we can re-sort and git blame ignore a fix at a later date.